packaging/Jenkinsfile

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/groovy
```

Comment:
Remove this file.  FWIW: `<path_to_packaging_project_dir>/install` can be used to update just the needed files in a project.